tcpdump.c

```diff
@@ -2056,7 +2056,7 @@ main(int argc, char **argv)
 
 		case OPTION_SKIP:
 			packets_skipped = parse_u_int("packet skip count",
```

Comment:
Maybe just make `packets_skipped` an `int`, and fetch it with `parse_int()`?

(And maybe rename it as `packets_to_skip`?)

Reply:
Yes, leave it as is for now.  In the long run, all counts should be unsigned and 64-bit, and the capture/read API should have callback routines that return a value, so they can report errors, do their own counting and say "the loop is done", etc..